[2022-11-14T04:53:14.230+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.is_api_availible scheduled__2022-11-13T00:00:00+00:00 [queued]>
[2022-11-14T04:53:14.735+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.is_api_availible scheduled__2022-11-13T00:00:00+00:00 [queued]>
[2022-11-14T04:53:14.750+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2022-11-14T04:53:14.756+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2022-11-14T04:53:14.761+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2022-11-14T04:53:15.004+0000] {taskinstance.py:1383} INFO - Executing <Task(HttpSensor): is_api_availible> on 2022-11-13 00:00:00+00:00
[2022-11-14T04:53:15.154+0000] {standard_task_runner.py:55} INFO - Started process 148 to run task
[2022-11-14T04:53:15.331+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'is_api_availible', 'scheduled__2022-11-13T00:00:00+00:00', '--job-id', '96', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmpj49zkfot']
[2022-11-14T04:53:15.413+0000] {standard_task_runner.py:83} INFO - Job 96: Subtask is_api_availible
[2022-11-14T04:53:17.383+0000] {task_command.py:376} INFO - Running <TaskInstance: user_processing.is_api_availible scheduled__2022-11-13T00:00:00+00:00 [running]> on host bca668ad4dbc
[2022-11-14T04:53:18.932+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=is_api_availible
AIRFLOW_CTX_EXECUTION_DATE=2022-11-13T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-11-13T00:00:00+00:00
[2022-11-14T04:53:18.956+0000] {http.py:120} INFO - Poking: api/
[2022-11-14T04:53:19.046+0000] {base.py:71} INFO - Using connection ID 'user_api' for task execution.
[2022-11-14T04:53:19.099+0000] {http.py:148} INFO - Sending 'GET' to url: https://randomuser.me/api/
[2022-11-14T04:53:20.616+0000] {base.py:213} INFO - Success criteria met. Exiting.
[2022-11-14T04:53:20.706+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=is_api_availible, execution_date=20221113T000000, start_date=20221114T045314, end_date=20221114T045320
[2022-11-14T04:53:21.134+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2022-11-14T04:53:21.668+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
